app/screens/BookDetail/components/BookDetailOptions/index.js

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
 import { View, Text, Image } from 'react-native';
-
-import { bookDataPropType } from '../../../../../propTypes';
-import defaultImg from '../../../../../assets/no_image.jpeg';
-import Button from '../../../../components/baseComponents/Button';
-import { lightBlue, white } from '../../../../../scss/colors';
+// eslint-disable-next-line import/extensions
```

Comment:
instead of using `next-line`, why don't you use it next to the import with only `line`?